src/main/resources/jenkins/plugins/extracolumns/BuildDurationColumn/columnHeader.jelly

```diff
@@ -31,5 +31,8 @@ THE SOFTWARE.
         <j:when test="${it.getBuildDurationType()==1}">
             <th>${%Average Duration}</th>
         </j:when>
+        <j:when test="${it.getBuildDurationType()==2}">
+            <th title="${%Average duration in minutes (compact)}">⏳'</th>
```

Comment:
At least in Jenkins version 1.580 there is no support for Unicode/Emojis. So I'd recommend to add the image instead.